stable/datadog/Chart.yaml

```diff
@@ -13,3 +13,5 @@ sources:
 maintainers:
 - name: Greg Taylor
   email: gtaylor@gc-taylor.com
+- name: Haissam Kaj
```

Comment:
I don't have the bandwidth to review the rest of this, but wanted to give this the :+1: as the current and only maintainer. The situation should improve greatly in having upstream involved!